resources/application-connector/values.yaml

```diff
@@ -42,31 +42,31 @@ global:
       version: "PR-13670"
     application_connectivity_certs_setup_job:
       name: "application-connectivity-certs-setup-job"
-      version: "PR-13170"
+      version: "708f6a87"
     application_connectivity_validator:
       name: "application-connectivity-validator"
-      version: "PR-13170"
+      version: "cc89c542"
     application_gateway:
       name: "application-gateway"
-      version: "PR-13171"
+      version: "6d430445"
     application_operator:
       name: "application-operator"
-      version: "PR-12257"
+      version: "3f163e8f"
     application_registry:
       name: "application-registry"
-      version: "PR-12257"
+      version: "a0a4ec40"
     central_application_connectivity_validator:
       name: "central-application-connectivity-validator"
-      version: "PR-12963"
+      version: "708f6a87"
```

Comment:
PR `PR-12963` is merged as a commit [245170b](https://github.com/kyma-project/kyma/commit/245170b15a885fdca5c658c0619dcab00f39c3bc). Therefore, the bumped image should be `245170b1`